core/src/main/java/org/mule/expression/DefaultExpressionManager.java

```diff
@@ -597,7 +597,11 @@ public boolean isValidExpression(String expression)
         }
         catch (InvalidExpressionException e)
         {
-            logger.warn(e.getMessage());
+            logger.warn("Expression '" + expression + "' is invalid: " + e.getMessage());
```

Comment:
This will result in 2 almost identical log lines in some scenarios. I think it's best to just keep the debug one.

Reply:
Fine for me.